src/webui/www/translations/webui_translations.qrc

```diff
@@ -0,0 +1,54 @@
+<RCC>
+    <qresource prefix="/www">
```

Comment:
Just tested, the i18n in webUI fails now.
Fixed by changing it to:
```xml
<RCC>
    <qresource prefix="/www/translations">
        <file>webui_ar.qm</file>
        ...
    </qresource>
</RCC>

```

Reply:
Thank you!